crates/fj/src/abi/context.rs

```diff
@@ -47,9 +61,17 @@ impl crate::models::Context for Context<'_> {
 
             let name = StringSlice::from_str(name);
 
-            match get_argument(user_data, name).into_str() {
-                "" => None,
-                other => Some(other),
+            match name.trim().is_empty() {
+                true => None,
+                false => match get_argument(user_data, name) {
+                    super::ffi_safe::Result::Ok(other) => {
+                        match other.is_empty() {
+                            true => None,
+                            false => Some(other.into_str()),
+                        }
+                    }
+                    super::ffi_safe::Result::Err(_) => None,
```

Comment:
It's not ideal to swallow the error here, as that could cause bugs to be hidden. Instead of getting an error message, making it clear that a bug has occurred, it could lead to a specified argument just not being recognized, which would be very confusing.

I think `Context::get_argument` should also return `Result` instead of an `Option`.